### PR TITLE
feature #163730056 A user has dashboard with scheduled meetup

### PR DESCRIPTION
**Pull request by abidex4yemi**
### What does this PR do?

Allow a user to see all activity within the questioner app after a successful login by showing total posted questions, total commented questions and their scheduled upcoming meetup.

### Description of the task to be completed?

   * Have the following listed task completed

      - connect user dashboard front-end page with the listed API endpoints below:

     `/api/v1/meetups/rsvps to get all user scheduled meetup`

     `/api/v1/questions/user to get total user posted questions`

     `/api/v1/comments/user to get total user commented questions`

### How should this be tested manually?
  * By visiting the link listed below

     [Meetup-questioner-app](https://abidex4yemi.github.io/meet-up-questioner/sign-in.html)

### What are the relevant pivotal tracker stories?

<a href='https://www.pivotaltracker.com/story/show/163730056'>#163730056</a>
<a href='https://www.pivotaltracker.com/story/show/163742231'>#163742231</a>
<a href='https://www.pivotaltracker.com/story/show/163751741'>#163751741</a>
user-dashboard-1

### CODE REVIEW

File: UI/js/user-dashboard.js [hunk 1]
```diff
@@ -12,3 +12,171 @@ if (localStorage.getItem('user')) {
...
+};
+
+/**
+ * Get total number of posted questions by user
+ */
+const getTotalCommentedQuestions = () => {
+  // comments/user endpoint url
+  const url = 'https://meet-up-questioner.herokuapp.com/api/v1/comments/user';
+
+  // make a GET request to questions/user
```
Comment (JCanaks) [inline UI/js/user-dashboard.js]: @abidex4yemi The comment you have here is misleading should be `// make a GET request to comments/user`

File: app/controllers/MeetupController.js [hunk 2]
```diff
@@ -109,6 +109,41 @@ class MeetupController {
...
+  /**
+   * get all users scheduled upcoming meetup
+   *
+   * @param {*} req
+   * @param {*} res
+   */
+  static async getAllRsvps(req, res) {
+    try {
+      // Get a single meet up record
+      const queryString = 'SELECT meetups.topic, meetups.location, meetups.happeningon, meetups.tags FROM meetups LEFT JOIN rsvps ON meetups.id = rsvps.meetupid WHERE rsvps.userid = $1 AND rsvps.response = $2';
```
Comment (JCanaks) [inline app/controllers/MeetupController.js]: @abidex4yemi what happens if the meetup date has passed?

File: UI/js/user-dashboard.js (hunk 1 above)
Comment (abidex4yemi) [reply UI/js/user-dashboard.js]: fixed

File: app/controllers/MeetupController.js (hunk 2 above)
Comment (abidex4yemi) [reply app/controllers/MeetupController.js]: Fixed

**Pull request merged**